scripts/publish-site.sh

```diff
@@ -11,6 +11,11 @@ yarn
 echo "=== Copying built Gatsby to website."
 gatsby-dev --scan-once --quiet
 
+echo "=== find graphql packages"
+
+find node_modules -name graphql
+echo "=== END find graphql packages"
```

Comment:
i am hoping this is better after the next npm publish since everything has been moved to peer deps :/